src/app/lib/utilities/preprocessor/rules/cpsAssetPage/addBylineBlock/models/index.js

```diff
@@ -0,0 +1,8 @@
+const convertToBylineBlock = block => ({
+  model: {
+    blocks: [block],
+  },
+  type: 'byline',
+});
```

Comment:
Is there any reason the logic here isn't included in https://github.com/bbc/simorgh/pull/5408/files#diff-5eb052837057f6af416734b84f5cfd25R6-R18 `convertToBylineBlock`?

Reply:
@andrewscfc there's a convention in most of the preprocessor rules to separate out the block conversion logic in a `/models` folder. I was just following that, but I think in this case we could move it back.

Reply:
Happy to stick to convention if it has value. So I guess the logic here is transformation logic: https://github.com/bbc/simorgh/pull/5408/files#diff-5eb052837057f6af416734b84f5cfd25R6-R18 while the logic in `src/app/lib/utilities/preprocessor/rules/cpsAssetPage/addBylineBlock/models/index.js ` is 'block conversion logic' ?